semeio/workflows/localisation/local_config_script.py

```diff
@@ -4,38 +4,45 @@
 # Current version:
 #   Localisation configuration file in YAML format is read for
 #   parameters and this script setup the localisation in ERT.
+from ert_shared.libres_facade import LibresFacade
+from ert_shared.plugins.plugin_manager import hook_implementation
 
 import semeio.workflows.localisation.local_script_lib as local
-
-from ert_shared.plugins.plugin_manager import hook_implementation
 from semeio.communication import SemeioScript
 from semeio.workflows.localisation.localisation_config import LocalisationConfig
 
 
 class LocalisationConfigJob(SemeioScript):
     def run(self, *args):
         ert = self.ert()
-
+        facade = LibresFacade(self.ert())
         # Clear all correlations
         local.clear_correlations(ert)
 
         # Read yml file with specifications
         config_dict = local.read_localisation_config(args)
 
         # Get all observations from ert instance
-        ert_obs_list = local.get_observations_from_ert(ert)
+        obs_keys = [
+            facade.get_observation_key(nr)
+            for nr, _ in enumerate(facade.get_observations())
+        ]
 
-        ert_param_dict, ert_node_type_dict = local.get_param_from_ert(ert)
+        ert_parameters = local.get_param_from_ert(ert.ensembleConfig())
```

Comment:
If possible it is always best to send in variables to functions as opposed to unpacking them on the function side. That way the functions have to know a minimum about the structure of the input.